doc/nrf/templates/sample_README.rst

```diff
@@ -61,34 +84,132 @@ Connect PIN1 to PIN2, then cut the connection again.
 
 User interface*
```

Comment:
Can we also include CLI commands and UART commands in this section? Not exactly hardware pieces of interface, but they serve a similar purpose.

See the following samples for examples:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/ncp/README.html
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/cli/README.html